src/EditorFeatures/Core/Organizing/OrganizeDocumentCommandHandler.cs

```diff
@@ -115,61 +100,107 @@ private static bool IsCommandSupported(EditorCommandArgs args, bool needsSemanti
             return false;
         }
 
-        public bool ExecuteCommand(SortImportsCommandArgs args, CommandExecutionContext context)
+        private bool ExecuteCommand(
+            EditorCommandArgs commandArgs,
+            CommandExecutionContext context,
+            Func<ITextSnapshot, IUIThreadOperationContext, Task<Document?>> getCurrentDocumentAsync,
+            Func<Document, CancellationToken, Task<Document>> getChangedDocumentAsync)
```

Comment:
created a simple helper to do all the work.  the three command calls in with tehir varying ways of getting the current document, and then manipulating it.